system/lib/libc/musl/src/thread/__timedwait.c

```diff
@@ -33,18 +32,20 @@ static int __futex4_cp(volatile void *addr, int op, int val, const struct timesp
 	if (r != -ENOSYS) return r;
 	return __syscall_cp(SYS_futex, addr, op & ~FUTEX_PRIVATE, val, to);
 }
-#endif
 
 static volatile int dummy = 0;
 weak_alias(dummy, __eintr_valid_flag);
+#endif
 
 int __timedwait_cp(volatile int *addr, int val,
 	clockid_t clk, const struct timespec *at, int priv)
 {
 	int r;
 	struct timespec to, *top=0;
 
+#ifndef __EMSCRIPTEN__
 	if (priv) priv = FUTEX_PRIVATE;
```

Comment:
Why this?

Reply:
Unnecessary code size optimization change, reverted with commit 6ea3ad394b1b870f157be7162b7546f64a8896e3.